README.md

```diff
@@ -213,6 +215,65 @@ into the repository. If you have carefully organized your commits and
 believe they should be merged without squashing, please mention it in
 a comment.
 
+### Licensing and Compliance Requirements
+
+Please review our [Open Source Policy][11] for complete guidelines on licensing
+and compliance. Below is a summary of the key points affecting
+**all external contributors**:
+
+- **Accepted Licenses:** Any code contributed must be licensed under the
```

Comment:
I think we can remove the bolding from the list headers. We don't use them anywhere afaik. :)